stl/inc/random

```diff
@@ -2782,10 +2782,13 @@ public:
         void _Init(_Ty1 _Mean0) noexcept { // set internal state
             _STL_ASSERT(0.0 < _Mean0, "invalid mean argument for poisson_distribution");
             _Mean = _Mean0;
+            if (_Mean0 < 12.0) {

```

Comment:
This constant is now used twice. Maybe it would be good to have either `_Is_small` member function or a named constant, so that `12.0` value is spelled out once?

Reply:
I'd be fine with a named constant here. I'm also fine with the code as-is since the scope of the magic is fairly small, `12.0` looks pretty unique (I'd be more concerned if it were like `1.0` which blends in with everything else), and it occurs only twice.